src/ORM/Connect/MySQLiConnector.php

```diff
@@ -81,15 +82,21 @@ public function connect($parameters, $selectDB = false)
         // Connection charset and collation
         $connCharset = Config::inst()->get(MySQLDatabase::class, 'connection_charset');
         $connCollation = Config::inst()->get(MySQLDatabase::class, 'connection_collation');
+        $socket = Environment::getEnv('SS_DATABASE_SOCKET');
+        $flags = Environment::getEnv('SS_DATABASE_FLAGS');
+
+        $flags = $flags ? array_reduce(explode(',', $flags), function ($carry, $item) {
+            $item = trim($item);
+            return $carry | constant($item);
+        }, 0) : $flags;
```

Comment:
This isn't super easy to read but I won't ask for you to change it now. Just something to think about for next time.